### PR TITLE
docs(samples): Retail API. Setup teardown resources scripts

### PR DESCRIPTION
**Pull request by t-karasova**

### CODE REVIEW

File: retail/interactive-tutorial/TestResourcesSetupCleanup/RemoveTestResources.cs
Comment (amanda-tarafa) [inline retail/interactive-tutorial/TestResourcesSetupCleanup/RemoveTestResources.cs:144]: Oh please don't reuse this attribute mechanism to run the setup. Is this somethig you expect tutorial users to have to run as well?

File: retail/interactive-tutorial/TestResourcesSetupCleanup/Attributes/ExampleAttribute.cs
Comment (amanda-tarafa) [reply retail/interactive-tutorial/TestResourcesSetupCleanup/Attributes/ExampleAttribute.cs:1]: Oh I see, you are using the same mechanism for running the setup as you are doing for the samples.

If this setup something that users will have to execute themselves for tutorials?

In any case, please do not repeat the code, create a separate project (Runner) for the attribute and the helper and you can depend on it here and in the Samples project. You can also add the code that you have in both Programs.cs (here and in the samples project) to the Runner project, for instance by adding a method RunScript(string[] args) and the you call that method from the main methods of this setup project and the Samples project.

File: retail/interactive-tutorial/TestResourcesSetupCleanup/TEST_RESOURCES_SETUP_CLEANUP.md
Comment (amanda-tarafa) [inline retail/interactive-tutorial/TestResourcesSetupCleanup/TEST_RESOURCES_SETUP_CLEANUP.md:1]: Change the name of this file to simply README.md

File: retail/interactive-tutorial/TestResourcesSetupCleanup/TestResourcesSetupCleanup.csproj
Comment (amanda-tarafa) [inline retail/interactive-tutorial/TestResourcesSetupCleanup/TestResourcesSetupCleanup.csproj:5]:
```suggestion
    <TargetFramework>netcoreapp3.1</TargetFramework>
```

File: retail/interactive-tutorial/TestResourcesSetupCleanup/TestResourcesSetupCleanup.csproj
Comment (amanda-tarafa) [inline retail/interactive-tutorial/TestResourcesSetupCleanup/TestResourcesSetupCleanup.csproj:9]:
```suggestion
```

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
OK, please add a Runner project as per my comment on the attribute file on a separate PR. That one can be reused by both the setup and the samples.
And once that's merged, modify this PR and #1602 so that the setup and samples project depend on the Runner project.

I will try to run the setup now on my machine so that we can start checkick the tests of the Samples project. I haven't reviewed the actual setup and cleanup code, I'll do that after the Runner has been extracted as I think that will make things cleaner.

File: retail/interactive-tutorial/TestResourcesSetupCleanup/RemoveTestResources.cs
Comment (t-karasova) [reply retail/interactive-tutorial/TestResourcesSetupCleanup/RemoveTestResources.cs:144]: no, this code is not for users, just for the person who will support the tests

Comment (t-karasova) [reply retail/interactive-tutorial/TestResourcesSetupCleanup/RemoveTestResources.cs:144]: the attribute mechanism will be removed

File: retail/interactive-tutorial/TestResourcesSetupCleanup/TEST_RESOURCES_SETUP_CLEANUP.md
Comment (t-karasova) [reply retail/interactive-tutorial/TestResourcesSetupCleanup/TEST_RESOURCES_SETUP_CLEANUP.md:11]: sure, I will fix it

**Comment by amanda-tarafa** [comment]
I've ran the setup on the project and data should be available for running tests.

**Comment by amanda-tarafa** [comment]
Note: We are going to focus on the samples and tutorials PRs first as that has higher priority, and once that's all in we can focus on this one. The test data has already been created on the test project and that's the only inmediate use we had for this PR.
I've marked as `do not merge` and `Draft` for that reason.

File: retail/interactive-tutorial/TestResourcesSetupCleanup/CreateTestResources.cs
Comment (kweinmeister) [inline retail/interactive-tutorial/TestResourcesSetupCleanup/CreateTestResources.cs:1]: Please use 2022 on all headers

**Review by kweinmeister — CHANGES_REQUESTED** [review]

**Comment by amanda-tarafa** [comment]
@t-karasova Can this PR be closed now that we have #1699 ?

Thanks!

**Comment by mkoshurnikov** [comment]
Can be closed because it's outdated.

**Pull request closed without merging**
